docs/analytics_tools/python_libraries.md

```diff
@@ -13,26 +13,32 @@ kernelspec:
   name: python3
 ---
 (python-libraries)=
+
 # Useful Python Libraries
-The following libraries are available and recommended for use by Cal-ITP data analysts. Our JupyterHub environment comes with all of these installed already!
+
+The following libraries are available and recommended for use by Cal-ITP data analysts. Our JupyterHub environment comes with all of these installed already, except for `calitp-data-infra`.
 
 ## Table of Contents
+
 1. [shared utils](#shared-utils)
-1. [calitp-data-analysis](#calitp-data-analysis)
-1. [siuba](#siuba)
+2. [calitp-data-analysis](#calitp-data-analysis)
+3. [siuba](#siuba)
 <br> - [Basic Query](#basic-query)
 <br> - [Collect Query Results](#collect-query-results)
 <br> - [Show Query SQL](#show-query-sql)
 <br> - [More siuba Resources](more-siuba-resources)
-1. [pandas](pandas-resources)
-1. [Add New Packages](#add-new-packages)
+4. [pandas](pandas-resources)
+5. [Add New Packages](#add-new-packages)
+<br> - [calitp-data-infra](#calitp-data-infra)
```

Comment:
I grouped it under Add New Packages because the docs refer to the other packages as built into the environment, and I thought it would be useful to have the final section list other useful (but not automatically available) packages as subheadings. However, maybe I'm overemphasizing the bit of the docs about how the other packages come preinstalled on JupyterHub, and should just treat `cal-itp-data-infra` as another top-level heading? Happy to do what you feel makes the most sense.

Reply:
Yeah, I would suggest moving `calitp-data-infra` into an appendix. It really should be separate and unnecessary to install in the image for the most part.